plugins/woocommerce-admin/client/payments/payment-settings-banner.tsx

```diff
@@ -0,0 +1,190 @@
+import './payment-recommendations.scss';
+
+export const PaymentMethodsIcons = () => (
+	<div className="woocommerce-recommended-payments-banner__footer_icon_container">
+		<Visa />
+		<MasterCard />
+		<Amex />
+		<ApplePay />
+		<GooglePay />
+		<CB />
+		<Giropay />
+		<DinersClub />
+		<Discover />
+		<UnionPay />
+		<JCB />
+		<Sofort />
+	</div>
+);
+
+const WcPayBanner = () => {
+	const WC_PAY_SETUP_URL =
+		'./admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments';
+	return (
+		<Card size="medium" className="woocommerce-recommended-payments-banner">
+			<CardBody className="woocommerce-recommended-payments-banner__body">
+				<div className="woocommerce-recommended-payments-banner__image_container">
+					<WCPayBannerImage />
+				</div>
+				<div className="woocommerce-recommended-payments-banner__text_container">
+					<Text
+						variant="title.small"
+						as="p"
+						size="24"
+						lineHeight="28px"
+						padding="0 20px 0 0"
+					>
+						{ __(
+							'Accept Payments and manage your business.',
+							'woocommerce'
+						) }
+					</Text>
+					<Text
+						className={
+							'woocommerce-recommended-payments__header-heading'
+						}
+						variant="caption"
+						as="p"
+						size="12"
+						lineHeight="16px"
+					>
+						{ __(
+							'By using WooCommerce Payments you agree to be bound by our Terms of Service and acknowledge that you have read our Privacy Policy',
+							'woocommerce'
+						) }
+					</Text>
+					<Button href={ WC_PAY_SETUP_URL } isPrimary>
+						{ __( 'Get started', 'woocommerce' ) }
+					</Button>
+				</div>
+			</CardBody>
+			<CardFooter className="woocommerce-recommended-payments-banner__footer">
+				<div>
+					<Text variant="caption" as="p" size="12" lineHeight="16px">
+						{ __(
+							'Accepted payment methods include:',
+							'woocommerce'
+						) }
+					</Text>
+				</div>
+				<div>
+					<PaymentMethodsIcons />
+				</div>
+				<div>
+					<Text variant="caption" as="p" size="12" lineHeight="16px">
+						{ __( '& more.', 'woocommerce' ) }
+					</Text>
+				</div>
+			</CardFooter>
+		</Card>
+	);
+};
+
+const DefaultPaymentMethodsHeaderText = () => (
+	<>
+		<h2>Payment Methods</h2>
+		<div id="payment_gateways_options-description">
+			<p>
+				Installed payment methods are listed below and can be sorted to
+				control their display order on the frontend.
+			</p>
+		</div>
+	</>
+);
+
+export const PaymentsBannerWrapper = () => {
+	const {
+		installedPaymentGateways,
+		paymentGatewaySuggestions,
+		isResolving,
+	} = useSelect( ( select: WCDataSelector ) => {
+		return {
+			installedPaymentGateways: select(
+				PAYMENT_GATEWAYS_STORE_NAME
+			).getPaymentGateways(),
+			isResolving: select( ONBOARDING_STORE_NAME ).isResolving(
+				'getPaymentGatewaySuggestions'
+			),
+			paymentGatewaySuggestions: select(
+				ONBOARDING_STORE_NAME
+			).getPaymentGatewaySuggestions(),
+		};
+	} );
+
+	const supportsWCPayments =
```

Comment:
Looks like `payment-recommendations.tsx` also has the same code. How about creating a util function for this?

https://github.com/woocommerce/woocommerce/blob/6151f4021c7b0beb9b2effcb41fdb6b18350655b/plugins/woocommerce-admin/client/payments/payment-recommendations.tsx#L84-L94

Reply:
Yea good call! I was planning to extract the entire hook and everything but ended up realising I only needed a small part of the useSelect and forgot that there was this part below that's duplicated

Reply:
Updated in 11f321a334220dd296e999426010702221aaba73